### DISCUSSION TITLE
Is there a way to show info with the warning on VSCode extension?

### DISCUSSION THREAD

**Discussion by mehm8128** [biomejs/biome#1390] [Q&A] [answered]
When I use React hooks such as `useEffect`, `useExhaustiveDependencies` lint rule shows warning such as `This hook specifies more dependencies than necessary.`. When I use biome on CLI, with the warnings, biome shows information such as `This dependency can be removed from the list.` with underline under the variables which is unnecessary, but on VSCode, biome only shows warning without information that what variable is unnecessary.
Is there a way to show such info with the warnings on VSCode? If such a way does't exist, could you implement the feature?

**Comment by ematipico** [comment]
It's possible that the VSCode extension is failing to resolve the configuration file.

Usually, that's caused by opening a project where `biome.json` isn't at the root of the project.

I believe there's an issue to track this, although it's unclear to us what's a proper solution

**Comment by mehm8128** [comment]
On my VSCode, error is shown as below. I want biome to show unnecessary variable name `currentWork`, or underline under `currentWork` variable in the deps array.
How the error and info are shown on your VSCode?
![image](https://github.com/biomejs/biome/assets/83744975/b14bae26-b4d5-4cd0-b245-67b5e822220a)

(sorry, I should have asked on https://github.com/biomejs/biome-vscode/discussions)

**Reply by ematipico** [reply, marked as answer]
Oh I see what you mean.

LSP diagnostics can contain only a subset of information, compared to Biome diagnostics.

So when we do convert those diagnostics, we only use the main message. Notes and advices aren't carried over.

**Reply by mehm8128** [reply]
Thank you.
`react-hooks/exhaustive-deps` rule in ESLint shows missing dependency such as `React Hook useEffect has a missing dependency: 'missingVariableName'. Either include it or remove the dependency array.`, but biome won't show such details?

**Reply by ematipico** [reply]
That's definitely an improvement that is worth pursuing. Feel free to open an issue, and if you want you can also contribute :)

**Reply by mehm8128** [reply]
OK, Thank you.
I'll mark this discussion as completed and open an issue.

**Reply by tcoopman** [reply]
For other people looking for this issue: https://github.com/biomejs/biome/issues/1391
